glide-core/tests/test_socket_listener.rs

```diff
@@ -44,6 +44,28 @@ mod socket_listener {
         ClosingError = 3,
     }
 
+    enum RedisType {
+        Cluster,
+        Standalone,
+    }
+
+    #[derive(Copy, Clone)]
+    enum Tls {
+        NoTls,
+        UseTls,
+    }
+
+    impl Tls {
+        fn to_bool(self) -> bool {
+            matches!(self, Tls::UseTls)
+        }
+    }
+
+    enum Sharing {
```

Comment:
'Sharing' still doesn't give enough info for readers - maybe TestServerType, or SharingServer?